src/routes/committee/announcements/+page.svelte

```diff
@@ -0,0 +1,162 @@
+<script lang="ts">
+	import {
+		Grid,
+		Row,
+		Column,
+		StructuredList,
+		StructuredListCell,
+		StructuredListRow,
+		Button,
+		Modal,
+		Form,
+		TextInput,
+		TextArea
+	} from 'carbon-components-svelte'
+	import type { announcement, newAnnouncement } from '$lib/types/domain/announcement'
+	import { Add, TrashCan } from 'carbon-icons-svelte'
+	import { addDoc, collection, deleteDoc, doc } from 'firebase/firestore'
+	import { page } from '$app/stores'
+	import { db } from '$lib/firebase/client/firebase'
+	import { Timestamp } from 'firebase/firestore'
+	import { newNewsPost } from '$lib/util/webhook'
+
+	export let data: { announcements: announcement[] }
```

Comment:
```suggestion
	export let data: PageData
```
En die moet je dan ff importen. Voordeel hiervan is dat je nooit mismatch tussen load result en page import hebt